internal/pkg/keeper/api/kv.go

```diff
@@ -0,0 +1,120 @@
+//
+// Copyright (C) 2024 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package api
+
+import (
+	"errors"
+	"net/http"
+	"net/url"
+	"path"
+
+	"github.com/edgexfoundry/go-mod-configuration/v3/internal/pkg/keeper/dtos"
+	httpUtils "github.com/edgexfoundry/go-mod-configuration/v3/internal/pkg/keeper/utils/http"
+)
+
+// KV is used to manipulate the K/V API
+type KV struct {
+	c *Caller
+}
+
+// KV is used to return a handle to the K/V apis
+func (c *Caller) KV() *KV {
+	return &KV{c}
+}
```

Comment:
Changing to`KeeperApi` with all API as suggested above cleans up the need for this.

Reply:
File removed, using the http clients from core-contract.